python/ray/data/_internal/planner/plan_download_op.py

```diff
@@ -189,8 +189,11 @@ def download_bytes_threaded(
         def load_uri_bytes(uri_path_iterator):
             """Function that takes an iterator of URI paths and yields downloaded bytes for each."""
             for uri_path in uri_path_iterator:
-                with fs.open_input_file(uri_path) as f:
-                    yield f.read()
+                try:
+                    with fs.open_input_file(uri_path) as f:
+                        yield f.read()
+                except OSError as _:
+                    yield None
```

Comment:
can you except and logging.debug the error / file path

this won't show by default but will be useful for debugging